### ISSUE TITLE
Webdav file upload error

### ISSUE THREAD

**Issue by 66Leo66** [RocketChat/Rocket.Chat#24128]
### Description:

Setup teracloud.jp webdav and failed to upload files.

### Steps to reproduce:

Setup webdav and upload file.

### Expected behavior:

Successful file upload.

### Actual behavior:

Top banner: `Error: Request failed with status code 401 [ufs: cannot upload file]`
Server log:
 ```
2022-01-10T10:45:06Z rocketchat-server.rocketchat-server[1387]: =================================
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: === UnHandledPromiseRejection ===
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: TypeError: Cannot read property 'logger' of null
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]:     at error (server/lib/logger/Logger.ts:97:8)
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]:     at /snap/rocketchat-server/1505/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:43:40
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: ---------------------------------
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: Errors like this can cause oplog processing errors.
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: Setting EXIT_UNHANDLEDPROMISEREJECTION will cause the process to exit allowing your service to automatically restart the process
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: Future node.js versions will automatically exit the process
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: =================================
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: === UnHandledPromiseRejection ===
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: TypeError: Cannot read property 'logger' of null
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]:     at error (server/lib/logger/Logger.ts:97:8)
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]:     at /snap/rocketchat-server/1505/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:43:40
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: ---------------------------------
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: Errors like this can cause oplog processing errors.
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: Setting EXIT_UNHANDLEDPROMISEREJECTION will cause the process to exit allowing your service to automatically restart the process
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: Future node.js versions will automatically exit the process
2022-01-10T10:45:14Z rocketchat-server.rocketchat-server[1387]: =================================
```
### Server Setup Information:

- Version of Rocket.Chat Server: 4.x/stabel from snap
- Operating System: Ubuntu 20.04 LTS
- Deployment Method: snap
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version:
- MongoDB Version:

### Client Setup Information94
- Desktop App or Browser Version: Chrome
- Operating System:Windows

**Comment by dudanogueira** [comment]
Hi!

Was this working before and stopped working?

It may be a configuration problem.

Also, please state the exact version you are using.

Can you try replicating this issue with a regular webdav server, using docker on latest version?

**Comment by dudanogueira** [comment]
I was able to reproduce this error.

I have used this as the webdav server:

  webdav:
    image: bytemark/webdav
    volumes:
      - /tmp/web:/var/lib/dav
    environment:
      - AUTH_TYPE=Basic
      - USERNAME=alice
      - PASSWORD=secret1234
      - LOCATION=/webdav
    ports:
      - 8011:80

**Comment by dougfabris** [comment]
Closing this issue, because it was fixed on this PR: #26422
Thanks for reporting it @66Leo66 if you still having trouble after updating it, please let me know!
